ipaserver/install/service.py

```diff
@@ -377,7 +379,36 @@ def add_cert_to_service(self):
             logger.critical("Could not add certificate to service %s entry: "
                             "%s", self.principal, str(e))
 
-    def import_ca_certs(self, db, ca_is_configured, conn=None):
+    def import_ca_certs_file(self, cafile, ca_is_configured, conn=None):
```

Comment:
I'm sorry, I just find the name of the method too confusing. Will call this couple `export_ca_certs_{file,nssdb}` in another fixup.